Assets/Plugins/ICP.Net/Bls.jslib

```diff
@@ -0,0 +1,6 @@
+var BLS = {
+  VerifySignature: function (publicKey, messageHash, signature) {
+    return blsVerify(publicKey, messageHash, signature);
```

Comment:
calls the blsVerify JS, but requires the bls.js script to be loaded in the html/browser